src/components/RestaurantDetails/components/RestaurantInfo/RestaurantInfo.js

```diff
@@ -0,0 +1,95 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import RestaurantInfoReader from './readers/RestaurantInfoReader';
+
+//Images
+import star from './images/star.svg';
+import delivery from './images/delivery.svg';
+import rupee from './images/rupee.svg';
+
+//Components
+import OtherInfo from './components/OtherInfo';
+
+//CSS
+import './RestaurantInfo.css';
+
+function RestaurantInfo(props) {
+  const { restaurantDetails } = props;
+  const name = RestaurantInfoReader.name(restaurantDetails);
+  const category = RestaurantInfoReader.category(restaurantDetails);
+  const area = RestaurantInfoReader.area(restaurantDetails);
+  const city = RestaurantInfoReader.city(restaurantDetails);
+  const deliveryTime = RestaurantInfoReader.deliveryTime(restaurantDetails);
+  const ratingValue = RestaurantInfoReader.ratingValue(restaurantDetails);
+  const ratingCount = RestaurantInfoReader.ratingCount(restaurantDetails);
+  const cost = RestaurantInfoReader.cost(restaurantDetails);
+  const noOfMembers = RestaurantInfoReader.noOfMembers(restaurantDetails);
+
+  return (
+    <div className='info'>
+      <h1>{name}</h1>
+      <p className='lightWeightText'>{category}</p>
+      <p className='lightWeightText'>
+        {area}, {city}
+      </p>
+      <div>
+        <OtherInfo
```

Comment:
choose a better name for this component something like RestaurantMetaInfo or Placeholder as we'd discussed